fms/add_sensor.py

```diff
@@ -0,0 +1,179 @@
+import subprocess
+
+from happi import Client
+from happi.errors import EnforceError, SearchError
+
+from .happi.containers import FMSRaritanItem, FMSSRCItem
+from .utils import TypeEnforcer as te
+
+fms_happi_database = "/cds/home/n/nrw/fms/db.json"
+sensor_name = 0
+
+
+def update_captar(first_name, middle_item, last_name, client=None):
+    if client is None:
+        client = Client(path=fms_happi_database)
+    first_item = client.find_item(name=first_name)
+    last_item = client.find_item(name=last_name)
+
+    if middle_item.captar_out != last_item.captar_in:
+        last_item.captar_in = middle_item.captar_out
+        middle_item.captar_in = first_item.captar_out
+    else:
+        first_item.captar_out = middle_item.captar_in
+        middle_item.captar_out = last_item.captar_in
+
+    middle_item.save()
+    last_item.save()
+    first_item.save()
+
+
+def find_port(name, client=None):
+    if client is None:
+        client = Client(path=fms_happi_database)
+    item = client.find_item(name=name)
+    return item.root_sensor_port
+
+
+def add_sensor_to_src(item, client=None):
```

Comment:
Suggestion: maybe some of these internals should have short docstrings that explain what they do and when to call them.

Reply:
I agree with everything Zach has commented here, but this is the one I'd like to echo the most.  Docstrings will help us remember what we're doing when we revisit this in the future.

Reply:
Added doc strings.